tests/evaluation/subsumption/subsumption.err

```diff
@@ -0,0 +1,18 @@
+Warning: Variable tail only occurs once in file subsumption.dl at line 85
```

Comment:
The line numbers for the expected error messages need to be updated to match the extra lines added for the license header.